src/main/cli/cli.c

```diff
@@ -6693,24 +6690,34 @@ static void processCharacter(const char c)
             if (cmd < cmdTable + ARRAYLEN(cmdTable)) {
                 cmd->cliCommand(cmd->name, options);
             } else {
-                cliPrintError("input", "UNKNOWN COMMAND, TRY 'HELP'");
+                if (cliInteractive) {
+                    cliPrintError("input", "UNKNOWN COMMAND, TRY 'HELP'");
+                } else {
+                    cliPrint("ERROR: ");
```

Comment:
Error is little bit too generic, it may be beter to differentiate between different errors
```suggestion
                    cliPrint("ERR_UNKNOWN: ");
```